### PR TITLE
feat: add user to gateway

### PR DESCRIPTION
**Pull request by dragosp1011**
### Context

- fixes #

### CODE REVIEW

File: packages/wallet/backend/src/gatehub/client.ts [hunk 1]
```diff
@@ -210,6 +246,22 @@ export class GateHubClient {
     return response
   }
 
+  async getWalletBalance(
+    userUuid: string,
+    walletId: string
+  ): Promise<IWalletBalance[]> {
+    const url = `${this.apiUrl}/core/v1/wallets/${walletId}/balances`
+
+    const response: IWalletBalance[] = await this.request<IWalletBalance[]>(
```
Comment (beniaminmunteanu) [inline packages/wallet/backend/src/gatehub/client.ts:255]: explicit typing annotation not needed, inference is correct by use of generics

File: packages/wallet/backend/src/gatehub/client.ts [hunk 2]
```diff
@@ -234,6 +286,25 @@ export class GateHubClient {
     return response
   }
 
+  async getRates(base: string): Promise<Record<string, number>> {
+    const url = `${this.apiUrl}/rates/v1/rates/current?counter=${base}&amount=1&useAll=true`
+
+    const response: IRatesResponse = await this.request<IRatesResponse>(
```
Comment (beniaminmunteanu) [inline packages/wallet/backend/src/gatehub/client.ts:292]: same nitpick ditto above

**Review by beniaminmunteanu — APPROVED** [review]
LGTM 🚀

**Review by sanducb — APPROVED** [review]
Looks good to me too! 👍

**Pull request merged**